modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -544,6 +544,18 @@ module.exports = (self) => {
     vueComponent: 'AposInputString',
     async convert(req, field, data, destination) {
       destination[field.name] = self.apos.launder.url(data[field.name], field.def, true);
+
+      if (field.required && (_.isUndefined(data[field.name]) || !data[field.name].toString().length)) {
```

Comment:
This will crash if the value is `null`. Use `data[field.name] == null`. Use of just two `=` is intentional, `== null` matches only `null` and `undefined` (a common pattern for this).